content/reference/user/profile/index.md

```diff
@@ -9,17 +9,18 @@ weight: 10030
 
 ## Accessing your profile
 
-You can view your profile at any time by clicking on the **My Profile** button in the navigation bar.
+{{< img src="profile_menu.png" alt="My profile button" >}}
 
-{{< img src="profile.png" alt="Profile" >}}
+You can view your profile at any time by clicking on the **My profile** button in the navigation bar.
+
+{{< img src="profile.png" alt="Profile view" >}}
 
 The information displayed are the following:
 
-- Username
 - First and Last name
-- Mail address
-- Different roles
+- Email address
+- Your roles (for all teams and for specific teams)
 
 To update your profile, modify the fields you want to change, then click on the button **Update your information**.
 
-You can't update your username here. You need to ask a System Admin to change your roles on the **Users Admin** page.
+You cannot change your GitHub username or your roles here. You need to ask a System Admin to change them in the **Users Admin** page.
```

Comment:
You could add something like this: To know who are your system admins, please check the organization page.